setup.py

```diff
@@ -125,6 +125,7 @@
     "jieba",
     "kenlm",
     "nltk",
+    "natten>=0.14.4",
```

Comment:
Same here, please don't add this dependency

Reply:
Please refer to my comment below.